File: 251229/Next Level/next-level.js
class User {
    constructor(id = 'codetree', level = 10) {
        this.id = id;
        this.level = level;
    }
}

const fs = require('fs');
const [id, level] = fs.readFileSync(0).toString().trim().split(' ');

const user1 = new User();
const user2 = new User(id, level);

console.log(`user ${user1.id} lv ${user1.level}`);
console.log(`user ${user2.id} lv ${user2.level}`);

File: 251229/Next Level/next-level.py
user2_id, user2_level = input().split()
user2_level = int(user2_level)

class User:
    def __init__(self, user_id = 'codetree', user_level = 10):
        self.user_id = user_id
        self.user_level = user_level

user1 = User()
user2 = User(user2_id, user2_level)

print(f'user {user1.user_id} lv {user1.user_level}')
print(f'user {user2.user_id} lv {user2.user_level}')